lib/driver.js

```diff
@@ -312,7 +312,7 @@ class XCUITestDriver extends BaseDriver {
     }
 
     if (this.isSimulator() && this.opts.udid && this.opts.customSSLCert) {
-      uninstallSSLCert(this.opts.customSSLCert, this.opts.udid);
+      await uninstallSSLCert(this.opts.customSSLCert, this.opts.udid);
```

Comment:
I'm actually not so sure whether we care to uninstall, but maybe it's good hygiene.